packages/cascara/src/ui/Section/__globals.js

```diff
@@ -0,0 +1,14 @@
+import pt from 'prop-types';
+import { PROP_TYPES as TitlePT } from '../Title/__globals';
+
+export const PROP_TYPES = {
+  /** HTML tag to allow polymorphism */
+  as: pt.string,
```

Comment:
@lordpixel @rociobaena technically, `as` should also allow a React component, which would not be a string. This can stay for now though as we are not doing that just yet. We still need to make a custom prop type for `as` so we can easily define it both in and outside of Cascara.

No change needed here either, just a heads up for us that we will want to replace all of these with our custom prop type when we create it. Keep defining this as `pt.string` for now.